### PR TITLE
A pull request for first capstone project

### PR DESCRIPTION
**Pull request by dlwub**
- Created a site for an International Career Fair 2022 event. 
- The home and about pages have been built (a desktop and mobile version for both pages)
- The home page has a link in the menu to the about page.
- The logo in the header links to the home page.
 - When a user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
 - The mobile menu has a close (X) button that closes the menu.
- The section "Featured speakers" has been created dynamically in JavaScript.
- A video description is prepared: https://www.loom.com/share/db883ca2917448b581c1060f9ee46b52.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,56 +1,41 @@
...
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JavaScript
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
```
Comment (Bondok6) [inline README.md:16]: - [ ] According to the requirements you should deploy your project using GitHub Pages, so kindly deploy it and put the link of the live demo in your `README` file 💯👌

File: README.md [hunk 2]
```diff
@@ -62,14 +47,17 @@ Feel free to check the [issues page](../../issues/).
...
-- etc
+- Microverse Community
 
-## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
+## 📝 License
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT.md) licensed.
```
Comment (Bondok6) [inline README.md:63]: - [ ] I can see you added `MIT.md` file, and you're using the Microverse template Great job 👌💯 but you forget to customize it, so kindly add your name and customize it 👌😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

image

File: about.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+mobileNavAbout.addEventListener('click', () => {
+  showMenu(aboutMenu, mobileNavAbout, closeBtnAbout, 
+  aboutMenuContainer, mobileMenuAbout);
+  });
+
+closeBtnAbout.addEventListener('click', () => {
+  hideMenu(mobileNavAbout, closeBtnAbout, aboutMenuContainer);
+  });
```
Comment (Bondok6) [inline about.js:40]: - [ ] In the mobile version, when I click on the hamburger menu, the mobile menu is not displayed, I can also notice that there are some errors in the `console`, so please fix these issues 👌

https://user-images.githubusercontent.com/45231276/183265250-635869d8-7874-414b-afbd-70eb3d9dd4fe.mp4

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @dlwub 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## To Highlight 
- No linter errors ✅
- Great job using semantic tags ✅
- Great job implementing the `Featured Speakers` section dynamically using js ✅
- Your repo is professional ✅

## Suggested changes

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Bondok6 — COMMENTED** [review]
# Note for the next reviewer 📝
### There's a Lighthouse linter error on GitHub but the rest of the linters [webhint, stylelint, eslint] are green, and when I installed the project and run the linters I didn't find any errors.

> So, please If there is no error when you run the checks, You can continue with the review. 👌

**Review by codecaiine — APPROVED** [review]
Hi @dlwub , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**